.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.linting.pylintEnabled": false
+}
```

Comment:
Is vscode your editor? Can you remote this setting file please.

We would expand ``.gitignore`` if that helps?